src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMModelMethods.scala

```diff
@@ -46,6 +46,22 @@ trait LightGBMModelMethods extends LightGBMModelParams {
     getLightGBMBooster.featuresShap(Vectors.sparse(size, indices, values))
   }
 
+  /** Public method to sets the start index of the iteration to predict.
+    * If <= 0, starts from the first iteration.
+    * @param startIteration The start index of the iteration to predict.
+    */
+  def setStartIteration(startIteration: Int): Unit = {
```

Comment:
Could you please try to make this a standard sparkml param so we don't need to add to python bindings? Thanks!

Reply:
done!